### DISCUSSION TITLE
Has the Light Protanopia & Deuteranopia theme changed since Apr 3?

### DISCUSSION THREAD

**Discussion by foundrytom** [community/community#51943] [Accessibility] [answered]
### Select Topic Area

Bug

### Body

Hey folks,

Since 4/4/23 green/red seems to have resurfaced in some UI states in the Light Protanopia & Deuteranopia theme. Notably status checks/text, and primary buttons. The buttons revert to blue on hover.

Is this intentional? It has been noted to reduce effectiveness in differentiating elements.

Many thanks.

![image](https://user-images.githubusercontent.com/12052093/229768463-f9f8dab7-1344-432a-a68e-ded035ec03bb.png)

Normal:
![image](https://user-images.githubusercontent.com/12052093/229769051-9dc1234a-10c1-43a9-b815-17c8f524ac10.png)
Hover:
![image](https://user-images.githubusercontent.com/12052093/229769193-ade0551b-4aef-4fae-8c9e-2992f4a24f77.png)

**Comment by lukasoppermann** [comment]
Thank you very much for the report. I will look into it immediately.

**Reply by lukasoppermann** [reply, marked as answer]
Hey @foundrytom this should be fixed now. Please let me know if you are still finding issues. Otherwise feel free to close this.

Thanks.

**Reply by foundrytom** [reply]
@lukasoppermann Thanks so much for sorting this so quickly 🙏  - confirmed fixed.
